ui-v2/app/routes/application.js

```diff
@@ -14,7 +14,7 @@ export default Route.extend(WithBlockingActions, {
   init: function() {
     this._super(...arguments);
   },
-  nspaceRepo: service('repository/nspace/disabled'),
+  nspacesRepo: service('repository/nspace/disabled'),
```

Comment:
Does this get overwritten by the initializer if namespaces are enabled?

Reply:
Ah yep, I see it above - maybe a comment here about that would be good just so it's not surprising.

Reply:
Good call, I think I figured the tests stuff :doh: Will add some comments here in a sec 👍